src/apps/watchfaces/OswAppWatchface.cpp

```diff
@@ -136,9 +136,12 @@ void OswAppWatchface::loop() {
     if (hal->btnHasGoneDown(BUTTON_3)) {
         hal->increaseBrightness(25);
     }
-    if (hal->btnHasGoneDown(BUTTON_2)) {
+     if (hal->btnDoublePress(BUTTON_2)) {
```

Comment:
is this for testing or do we want to change this?

Reply:
Just test :) plz check it next weekend !